src/Game.cpp

```diff
@@ -31,6 +31,164 @@
 #include "UnkObjects/unk6C2E88.hpp"
 #include "UnkObjects/unk6C3250.hpp"
 #include "UnkObjects/unk6C3890.hpp"
+#include "UnkObjects/unk8600F8.hpp"
+#include "config.hpp"
+
+static void sub_612A80(const char *args) {
+    glo_8600F0 = 0;
+    glo_8600EC = nullptr;
+    char *esi = (char*)args;
+    if (*esi != 0) {
+        esi = (char*)std::strchr(args, ' ');
```

Comment:
In this case though we need to be able to modify the args var so I'm going to change the definition to `char *`